services/movies-service.js

```diff
@@ -0,0 +1,38 @@
+const INITIAL_MOVIES = require('./movies.json')
+
+class MoviesService {
```

Comment:
thanks for the details explanation @nudelx. I am not convinced yet but maybe it's because I don't see the full picture. it's an interesting discussion I would like to do online with you.
I refactored the service to not use Class, but pure functions